ports/servoshell/minibrowser.rs

```diff
@@ -7,7 +7,7 @@ use std::num::NonZeroU32;
 use std::sync::Arc;
 use std::time::Instant;
 
-use egui::{CentralPanel, Frame, InnerResponse, Key, Modifiers, PaintCallback, TopBottomPanel};
+use egui::{CentralPanel, Frame, Key, Modifiers, PaintCallback, TopBottomPanel};
```

Comment:
Please remove this unrelated change.

Reply:
on my end : current branch its not there

Reply:
its only available on the main branch

Reply:
You need to rebase your change onto the main branch:

`git checkout main`
`git pull`
`git checkout wptreference`
`git rebase -i main`

Do the interactive rebase and then make sure that when you do `git diff main wptreference` the change is not there.

Reply:
Or I can simply rebase from the PR, I guess. I'll do that now.

Reply:
Great 
Thanks